tools/client-webpack/dev.js

```diff
@@ -7,7 +7,7 @@ const common = require('./common');
 const HtmlWebpackHarddiskPlugin = require('html-webpack-harddisk-plugin');
```

Comment:
path should be tools/build/client

Reply:
follow the same conventions for server too

Reply:
is it fine to have a separate config file for dev and prod? and will be merged with web pack-merge plugin utility.

Reply:
let's keep different file for prod & dev for simplicity